microsoft-graph.d.ts

```diff
@@ -8,7 +8,7 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.1
 
-export as namespace microsoftgraph;
+export as namespace microsoftgraphbeta;
```

Comment:
@nikithauc interesting, didn't we already have this in Monday's generation?

Reply:
yeah. I looking into this